options.rb

```diff
@@ -103,6 +139,32 @@ def add_album
     puts '----------------------------------------------'
   end
 
+  def add_book
+    print 'Publisher: '
+    publisher = gets.chomp
+
+    while true
+      print 'Cover state (select 1 for "good" or 2 for "bad" ): '
+      cover_option = gets.chomp.to_i
+
+      case cover_option
+      when 1 then cover_state = 'good'
+      when 2 then cover_state = 'bad'
+      else
+        puts "Wrong option \n\n"
+        next
+      end
+      break
+    end
+
+    print 'Publish date (YYYY-MM-DD): '
+    publish_date = gets.chomp
+
+    book = Book.new(publisher: publisher, cover_state: cover_state, publish_date: publish_date)
+    puts 'Book added successfully.'
```

Comment:
Please, before you tell the user that the book was added, make sure to add it! Right now if you add a book and then try to list them, a message saying 'no books available' appears. This should not be the case! You can add books to the list with this line: `@books << book`